src/schema/v1.json

```diff
@@ -872,17 +952,21 @@
    "name": {
      "$id":"#/properties/name",
      "type":"string",
-     "title":"The Name",
-     "default":"",
+     "title":"Name",
+     "default":"fedora-coreos",
      "examples": [
-       "rhcos"
+       "rhcos",
+       "fedora-coreos"
       ],
-     "pattern":"^(.*)$"
+     "enum": [
```

Comment:
It looks like this is exhaustive?  The whole idea is to support people doing custom builds, let's not force `name` to be one of those.

Reply:
Changed to support hack builds by defining a "core" schema with the expected outputs for RHCOS and FCOS.